src/utils/helpers.ts

```diff
@@ -0,0 +1,6 @@
+// Utility function to format string
+export const lowercaseHyphenateString = function (dirtyString: string) {
```

Comment:
Lets make this a arrow function for consistency. Should be something like `export const lowercaseHyphenateString = (dirtyString: string) => {`

Reply:
gotcha